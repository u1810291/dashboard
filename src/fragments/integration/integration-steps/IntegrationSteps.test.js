import React from 'react'
import { shallow } from 'enzyme'
import IntegrationSteps from '.'

describe('IntegrationSteps', () => {
  test('blocks render', () => {
    const component = shallow(<IntegrationSteps />)
    expect(component).toMatchSnapshot()
  })
})
