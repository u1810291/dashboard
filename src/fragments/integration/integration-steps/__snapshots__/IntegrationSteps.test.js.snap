// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntegrationSteps blocks render 1`] = `
Array [
  <Panel>
    <PanelBody
      className="mgi-items mgi-items--centered"
    >
      <section
        className="mgi-items--col-7"
      >
        <h2>
          <FormattedMessage
            id="settings.installationGuide.configure.title"
            values={Object {}}
          />
          <p>
            <FormattedMessage
              id="settings.installationGuide.configure.description"
              values={Object {}}
            />
          </p>
        </h2>
      </section>
      <Button
        buttonStyle="primary"
        className="mgi-items--col-5"
        href="/"
      >
        <FormattedMessage
          id="settings.installationGuide.configure.button"
          values={Object {}}
        />
      </Button>
    </PanelBody>
  </Panel>,
  <Panel>
    <PanelBody
      className="mgi-items mgi-items--centered"
    >
      <section
        className="mgi-items--col-7"
      >
        <h2>
          <FormattedMessage
            id="settings.installationGuide.install.title"
            values={Object {}}
          />
          <p>
            <FormattedMessage
              id="settings.installationGuide.install.description"
              values={Object {}}
            />
          </p>
        </h2>
      </section>
      <Button
        buttonStyle="primary"
        className="mgi-items--col-5"
        href="/integration/integration-code"
      >
        <FormattedMessage
          id="settings.installationGuide.install.button"
          values={Object {}}
        />
      </Button>
    </PanelBody>
  </Panel>,
  <Panel>
    <PanelBody
      className="mgi-items mgi-items--centered"
    >
      <section
        className="mgi-items--col-7"
      >
        <h2>
          <FormattedMessage
            id="settings.installationGuide.receive.title"
            values={Object {}}
          />
          <p>
            <FormattedMessage
              id="settings.installationGuide.receive.description"
              values={Object {}}
            />
          </p>
        </h2>
        <p>
          <a
            href="https://docs.getmati.com"
            rel="noopener noreferrer"
            target="_blank"
          >
            <FormattedMessage
              id="settings.installationGuide.receive.link"
              values={Object {}}
            />
          </a>
        </p>
      </section>
      <Button
        buttonStyle="primary"
        className="mgi-items--col-5"
        href="/integration/applications"
      >
        <FormattedMessage
          id="settings.installationGuide.receive.button"
          values={Object {}}
        />
      </Button>
    </PanelBody>
  </Panel>,
]
`;
